05-documentation.Rmd

```diff
@@ -0,0 +1,110 @@
+# Code Documentation
+
+## Introduction
+
+Documentation of code is essential, even if the only person using your code 
+is future-you. Use 
+[roxygen2](https://github.com/klutometis/roxygen) with enabled [markdown](https://github.com/klutometis/roxygen/blob/master/vignettes/markdown.Rmd)
+support to keep the documentation close to your code.
+
+```{r, include=FALSE}
+knitr::opts_chunk$set(eval = FALSE)
+```
+
+## Title and Description
+For the title, describe concisely what the function does at the very 
+top of your function documentation. Do not use a dot at the end of the title 
+and use title style capitalization. For both the title and the description, 
+there is no need to use the explicit `@title` or `@description` tag. To 
+visually separate the title and the description, leave one line blank after 
+each of them.
+```{r}
+#' Combine Values into a Vector or List
+#' 
+#' This is a generic function which combines its arguments.
+#'
+```
+
+## Indention
+Always indent with one space after `#'`. If any description corresponding to a 
+`roxygen` tag spans over multiple lines, add another two spaces of extra 
+indention.
+```{r}
+#' @param key The bare (unquoted) name of the column whose values will be used 
+#'   as column headings. 
+```
+
+Alternatively, tags that span over multiple lines (like `@examples`) can have 
+the corresponding tag on its own line and subsequent lines do not need extra
+indention.
+```{r}
+#' @examples
+#' 1 + 1
+#' sin(pi)
+```
+The section 'Description' does not need extra indention either.
+
+## Capitalization
+The first letter of every description corresponding to a `roxygen` tag is 
+upper case.
+```{r}
+#' @param key The bare (unquoted) name of the column whose values will be used 
+#'   as column headings. 
+```
+
+An exception to this rule are words in `code font` (see below), in which case
```

Comment:
This doesn't feel quite right here. I think you should same something more general like avoid capitalisation the names of functions or packages because R is case-sensitive and that would change the meaning.